api/src/pcapi/serialization/utils.py

```diff
@@ -134,9 +138,14 @@ def without_timezone(d: datetime.datetime) -> datetime.datetime:
     return datetime.datetime(d.year, d.month, d.day, d.hour, d.minute, d.second, d.microsecond)
 
 
-def check_date_in_future_and_remove_timezone(value: datetime.datetime | None) -> datetime.datetime | None:
+def check_date_in_future_and_remove_timezone(value: datetime.datetime | NOW_LITERAL | None) -> datetime.datetime | None:
     if not value:
         return None
+    if value == "now":
```

Comment:
Il me semblerait plus cohérent avec le fait de définir une constante d'avoir : 
```suggestion
    if value == NOW_LITERAL:
```
et peut-être cela permettrait-il de rendre mypy heureux sans besoin de '`assert` ?

Reply:
Je vais tester ça